.github/workflows/_build_bodo_conda_linux_comm.yml

```diff
@@ -53,6 +53,7 @@ jobs:
           # Conda-Build normally uses a random path. Setting these args disables that.
           # --croot for consistent base directory path for sccache
           # --build-id-pat for consistent build directory path for sccache
+          # Why `PLATFORM_CHANNEL` here?
```

Comment:
Yeah you're right, this is an old use, we don't need. Can you remove the use in the next line as well?